### DISCUSSION TITLE
Problem with COMB line.

### DISCUSSION THREAD

**Discussion by XuMu123** [firemodels/fds#13467] [Q&A]
I am a student of SCUT in China,I am study FDS.There is a question that I can not solve.
ERROR(101): Problem with COMB line. 
My COMB is N_SIMPLE_CHEMISTRY_REACTIONS=2, FUEL_C_TO_CO_FRACTION=1, FUEL_H_TO_H2_FRACTION=0.5 /

**Comment by drjfloyd** [comment]
One or more of your inputs are not valid for &COMB.  PLease consult the User's Guide for the version of FDS you are using and verify that all of your inputs on &COMB are allowed on &COMB, are spelled correctly, and have the correct type of data given.
